CONTRIBUTING.md

```diff
@@ -28,7 +28,7 @@ If you have questions about one of the issues, please comment on them, and one o
 
 Please provide as much context as possible when you open an issue. The information you provide must be comprehensive enough to reproduce that issue for the assignee. Therefore, contributors may use but aren't restricted to the issue template provided by the project maintainers.
 
-Please apply one or more applicable labels to your issue so that all community members are able to cluster the issues better. This applies especially for the Working Groups repository, please add the label of the Working Group to the issue.
+Please apply one or more applicable labels to your issue so that all community members are able to cluster the issues better.
```

Comment:
"Anyone with triage access to a repository can apply and dismiss labels" ... so an Outside Contributor with only read access can't do that. A consequence: Maintainers should get at least "triage" access.

Reply:
It makes sense to remove the sentence "Anyone with triage access to a repository can apply and dismiss labels" if it is in. I haven't found it...

Reply:
Sorry for the misunderstanding I produced. I propose to delete the sentence "Please apply one or more applicable labels to your issue so that all community members are able to cluster the issues better.", as a contributor with read access can't do it.

"Anyone with triage access to a repository can apply and dismiss labels" is a fact in GitHub, see here: https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels#applying-a-label

Reply:
Agree, sentence is deleted now